contrib/ansible/openshift/register.sh

```diff
@@ -68,10 +69,14 @@ while read -u 11 line;do
     pbench-register-tool --label=$label --name=pidstat --remote $remote
     pbench-register-tool --label=$label --name=oc --remote $remote
     pbench-register-tool --label=$label --name=pprof --remote $remote -- --osecomponent=master --interval=$ose_master_interval
+    pbench-register-tool --label=$label --name=haproxy-ocp --remote $remote -- --interval=$ose_master_interval --counters-clear-all
+    pbench-register-tool --label=$label --name=disk --remote $remote
   else
     pbench-register-tool --label=$label --name=sar --remote $remote
     pbench-register-tool --label=$label --name=iostat --remote $remote
     pbench-register-tool --label=$label --name=pidstat --remote $remote
+    pbench-register-tool --label=$label --name=disk --remote $remote
+    pbench-register-tool --label=$label --name=prometheus-metrics --remote $remote -- --inventory=$inventory_file_path
   fi
```

Comment:
You are doing a lot of the same things in the if part, as well as the else part. You should refactor the common cases out of the if/else and just deal with the differences in the if/else.